test/com/google/enterprise/adaptor/filenet/DocumentTraverserTest.java

```diff
@@ -582,6 +585,50 @@ public void testGetDocContent_allMetadataTypes() throws Exception {
         response.getMetadata());
   }
 
+  @Test
+  public void testGetDocContent_defaultAuthenticatedUsers() throws Exception {
+    testGetDocContent_authenticatedUsers("NT AUTHORITY\\Authenticated Users");
+  }
+
+  @Test
+  public void testGetDocContent_customAuthenticatedUsers() throws Exception {
+    options = TestObjectFactory.newConfigOptions(
+        ImmutableMap.<String, String>of(
+            "filenet.authenticatedUsersGroup",
+            "cn=AuthenticatedUsers,ou=users,dc=example,dc=com"));
+
+    testGetDocContent_authenticatedUsers("AuthenticatedUsers@example.com");
+  }
+
+  private void testGetDocContent_authenticatedUsers(String authenticatedUsers)
+      throws Exception {
+    String id = "{AAAAAAAA-0000-0000-0000-000000000000}";
+    DocId docId = newDocId(new Id(id));
+    MockObjectStore os = getObjectStore();
+    mockDocument(os, id, DOCUMENT_TIMESTAMP, RELEASED, 42d, "text/plain",
+        TestObjectFactory.newPermissionList(ImmutableList.of(
+            TestObjectFactory.newPermission(Permissions.AUTHENTICATED_USERS,
+                SecurityPrincipalType.GROUP, AccessType.ALLOW,
+                (AccessRight.READ_AS_INT | AccessRight.VIEW_CONTENT_AS_INT),
+                0, PermissionSource.SOURCE_DIRECT))));
+
+    DocumentTraverser traverser = new DocumentTraverser(options);
+    Request request = new MockRequest(docId);
+    RecordingResponse response = new RecordingResponse();
+    traverser.getDocContent(new Id(id), request, response);
+
+    Acl acl = response.getAcl();
+    assertEquals(acl.getPermitGroups().toString(),
```

Comment:
Don't need the message argument here with assertEquals.

Reply:
Done.